Assets.md

```diff
@@ -4,6 +4,9 @@ We use asset catalogs to organize our application’s assets.
 
 ### Key Considerations
 * Organize assets using folders inside asset catalogs.
-* Whenever possible, use images and colors from asset catalogs using Interface Builder.
-* When assets cannot be used from Interface Builder (i.e. the assets to use from the catalog are not known at compile time), make use of code generation solutions such as [SwiftGen](https://github.com/SwiftGen/SwiftGen#asset-catalog) to provide non-optional constants for assets.
+* All colors and images that are known at compile time should exist in asset catalogs. 
```

Comment:
Should you create colors that point to system colors or use the system colors directly?

Reply:
Good question. I’m inclined to say the latter, so "All colors and images that are known at compile time should exist in asset catalogs, unless they are system colors, in which case they do not need to be duplicated in the asset catalog." Any preference there, @Twigz?

Note this would also change the statement below to:

> Always set asset catalog and system colors in code…

or just

> Always set UI component colors in code…

Reply:
Updated according to the comment above, but would like @Twigz’ input

Reply:
Good to me!